srcpkgs/PWAsForFirefox/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'PWAsForFirefox'
+pkgname=PWAsForFirefox
+version=2.7.3
+revision=1
+archs="~*-musl"
```

Comment:
Any particular reason why musl is unsupported? Or do the builds just fail for musl for unknown reasons here while musl is generally supported?

Reply:
The author of the program lists glibc as a requirement. I don't know if musl would work anyway.

https://github.com/filips123/PWAsForFirefox/tree/main/native#requirements